lib/auth/auth.go

```diff
@@ -1629,6 +1650,21 @@ func certRequestDeviceExtensions(ext tlsca.DeviceExtensions) certRequestOption {
 	}
 }
 
+// getUserOrLoginState will return the given user or the login state associated with the user.
```

Comment:
Is there a legitimate case for the user *not* to have user logins state?

If not, we could force the user to re-login by deleting the user login state.

Reply:
Certificates issued directly with `tctl auth sign` (e.g. certs for a bot) won't have an associated login state.  There may be other cases as well.